blueprint/src/chapters/chapter1/sheet3.tex

```diff
@@ -0,0 +1,46 @@
+\section{Uniqueness Theorems}
+
+\begin{theorem}[Identity Uniqueness]
+    \label{theorem : identity_uniqueness}
+    \lean{AlgebraInLean.identity_uniqueness}
+    \leanok
+    \uses{definition : Monoid}
+    In a monoid, the identity element is unique:
+    $\forall a \in G, (\forall b \in G, \mu(a, b) = \mu(b, a) = b) \Rightarrow a = \mathbb{e}$
+  \end{theorem}
+
+\begin{theorem}[Inverse Uniqueness]
+    \label{theorem : inverse_uniqueness}
+    \lean{AlgebraInLean.inverse_uniqueness}
+    \leanok
+    \uses{definition : Group}
+    In a group, each element has a unique inverse:
+    $\forall a, b \in G, (\mu(a, b) = \mu(b, a) = \mathbb{e}) \Rightarrow b = \iota(a)$
```

Comment:
I'd use your words: if and then look better here than the lofical implication arrow.